endpoints/api/namespacequota.py

```diff
@@ -151,9 +151,9 @@ def get(self, orgname, quota_id):
     def put(self, orgname, quota_id):
         orgperm = AdministerOrganizationPermission(orgname)
 
-        # Only super users can update quota
         if not features.SUPER_USERS or not SuperUserPermission().can():
-            raise Unauthorized()
+            if not orgperm.can():
```

Comment:
I would suggest we remove super user permissions from within this and secure it using @require_scope(scopes.SUPERUSER) to be removed at a later time when the code is smarter